server/schedule/operator_controller.go

```diff
@@ -316,11 +320,13 @@ func (oc *OperatorController) PromoteWaitingOperator() {
 				operatorWaitCounter.WithLabelValues(op.Desc(), "promote_canceled").Inc()
 				_ = op.Cancel()
 				oc.buryOperator(op)
+				oc.wopStatus.ops[op.Desc()]--
```

Comment:
For merge, `GetOperator` will return two operators, but `oc.wopStatus.ops[op.Desc()]--` should only happen once.